### PR TITLE
TE-847: Expose strings props on `Reviews`

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-847)

### What **one** thing does this PR do?
Exposes the string props for `Reviews`

### Any other notes
- I have added a util called default-strings, following the same pattern as #273

__Preview__
screen shot 2018-08-20 at 12 27 38

### CODE REVIEW

File: src/components/property-page-widgets/Reviews/component.js [hunk 1]
```diff
@@ -10,16 +10,22 @@ import { Grid } from 'layout/Grid';
...
+  reviews,
+  ratingAverage,
+  reviewsLabel,
+  submitReviewLabel,
+}) => (
   <Grid>
     <GridRow>
       <GridColumn width={12}>
-        <Heading>Reviews</Heading>
+        <Heading>{reviewsLabel}</Heading>
```
Comment (Jonnygwi) [inline src/components/property-page-widgets/Reviews/component.js]: This needs to change to `reviewHeading`

File: src/components/property-page-widgets/Reviews/component.js [hunk 2]
```diff
@@ -99,4 +107,8 @@ Component.propTypes = {
       reviewerStayDate: PropTypes.string.isRequired,
     })
   ),
+  /** The reviews title */
```
Comment (Jonnygwi) [inline src/components/property-page-widgets/Reviews/component.js]: This comment should follow the same naming. We are using the word heading throughout the project not title

File: src/components/property-page-widgets/Reviews/component.js (hunk 1 above)
Comment (stephenkr) [reply src/components/property-page-widgets/Reviews/component.js]: Yep true, fixing now.

File: src/components/property-page-widgets/Reviews/component.js [hunk 3]
```diff
@@ -47,7 +53,7 @@ export const Component = ({ reviews, ratingAverage }) => (
         verticalAlign="middle"
       >
         <Button isCompact isPositionedRight isRounded size="medium">
-          Submit a review
+          {submitReviewLabel}
```
Comment (Jonnygwi) [inline src/components/property-page-widgets/Reviews/component.js]: This `submitReviewLabel` isn't really a label. It's more like `submitReviewButtonText`

**Pull request merged**